README.md

```diff
@@ -3,10 +3,20 @@
 Etherpass all-in-one web and desktop application.
 
 ## Getting started
-It is recommended to use [yarn](yarnpkg.com) as a package manager
```

Comment:
Could you plz revert back a description of the local development workflow with `yarn`? Since it's still possible along with VM/Vagrant-way